TwitchDownloaderCore/TwitchHelper.cs

```diff
@@ -264,11 +264,15 @@ public static async Task<List<TwitchEmote>> GetThirdPartyEmotes(int streamerId,
                 {
                     if (alreadyAdded.Contains(emote.Code))
                         continue;
-                    TwitchEmote newEmote = new TwitchEmote(await GetImage(bttvFolder, emote.ImageUrl.Replace("[scale]", "2"), emote.Id, "2", emote.ImageType), EmoteProvider.ThirdParty, 2, emote.Id, emote.Code);
-                    if (emote.IsZeroWidth)
-                        newEmote.IsZeroWidth = true;
-                    returnList.Add(newEmote);
-                    alreadyAdded.Add(emote.Code);
+                    try
+                    {
+                        TwitchEmote newEmote = new TwitchEmote(await GetImage(bttvFolder, emote.ImageUrl.Replace("[scale]", "2"), emote.Id, "2", emote.ImageType), EmoteProvider.ThirdParty, 2, emote.Id, emote.Code);
+                        if (emote.IsZeroWidth)
+                            newEmote.IsZeroWidth = true;
+                        returnList.Add(newEmote);
+                        alreadyAdded.Add(emote.Code);
+                    }
+                    catch { }
```

Comment:
I would personally use `catch (HttpRequestException ex) when (ex.StatusCode == HttpStatusCode.NotFound)` so that code 400 (bad request) and others can propagate upwards, rather than failing silently